File: CS/MvcDashboardUseDifferentCaches/Controllers/HomeController.cs
using System.Web.Mvc;

namespace MvcDashboardUseDifferentCaches.Controllers
{
    public class HomeController : Controller {
        public ActionResult Index() {
            return View();
        }
        public ActionResult Refresh() {
            CacheManager.ResetCache();
            return new EmptyResult();
        }
    }
}

File: VB/MvcDashboardUseDifferentCaches/Controllers/HomeController.vb
Imports System.Web.Mvc

Namespace MvcDashboardUseDifferentCaches.Controllers
	Public Class HomeController
		Inherits Controller

		Public Function Index() As ActionResult
			Return View()
		End Function
		Public Function Refresh() As ActionResult
			CacheManager.ResetCache()
			Return New EmptyResult()
		End Function
	End Class
End Namespace